agent/agent.go

```diff
@@ -1676,120 +1677,6 @@ func (a *Agent) setupNodeID(config *config.RuntimeConfig) error {
 	return nil
 }
 
-// setupBaseKeyrings configures the LAN and WAN keyrings.
-func (a *Agent) setupBaseKeyrings(config *consul.Config) error {
```

Comment:
These two functions were moved to `keyring.go`. The only changes to them were to remove the `a *Agent` and replace it with a `logger` argument.